code/game/objects/structures/signs/signs_departments.dm

```diff
@@ -209,6 +198,42 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/drop, 32)
 	sign_change_name = "Location - Courtroom"
 	desc = "A sign labelling the courtroom, where the ever sacred Space Law is upheld."
 	icon_state = "court"
-	is_editable = TRUE
 
 MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/court, 32)
+
+/obj/structure/sign/departments/telecomms
+	name = "\improper Telecommunications sign"
+	sign_change_name = "Location - Telecommunications"
+	desc = "A sign labelling an area where the station's radio and NTnet servers are stored."
+	icon_state = "telecomms"
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/telecomms, 32)
+
+/obj/structure/sign/departments/telecomms/alt
+	icon_state = "telecomms2"
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/telecomms/alt, 32)
+
+/obj/structure/sign/departments/aiupload
+	name = "\improper AI Upload sign"
+	sign_change_name = "Location - AI Upload"
+	desc = "A sign labelling an area where laws are uploaded to the station's AI and cyborgs."
+	icon_state = "aiupload"
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/aiupload, 32)
+
+/obj/structure/sign/departments/aisat
+	name = "\improper AI Sattelite sign"
+	sign_change_name = "Location - AI Sattelite"
+	desc = "A sign labelling a sattelite where the AI is installed."
```

Comment:
```suggestion
	desc = "A sign labelling a satellite where the AI is installed."
```

Spellcheck, also this just kinda reads odd to me? I dunno if there's a better way to phrase it, but I suppose this works.

Reply:
It's consistent with the other sign's "A sign labelling an (area type) where (thing happens)" format. The intent of these messages is to give a short blurb on what the indicated area is to new players, who are the audience of these signs. However even if it breaks format it isn't very readable so I'll update it.